lms/static/sass/discussion/_mixins.scss

```diff
@@ -63,7 +63,7 @@
   border-radius: 3px 3px 0 0;
   padding: ($baseline/2);
   width: 100%;
-  height: 240px;
+  height: 125px;
```

Comment:
Context: On focus of the response / comment input area, we change the height from 240px to 125px. This change sets the initial height of the input area to the same value it is forced to on focus. This eliminates unnecessary visual height for response and comment authoring, as well as the flicker / change users see when they click into this input area.